src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -1035,7 +1040,6 @@ public async Task GetOngoingTaskInfo()
                             {
                                 TaskId = sqlEtl.TaskId,
                                 TaskName = sqlEtl.Name,
-                                MentorNode = sqlEtl.MentorNode,
```

Comment:
Why remove this?

Reply:
@Danielle9897  We noticed that this property appears twice in all of the `ETLs` response objects.

Reply:
What is the reason for this?

Reply:
This is for example what is returned for an ETL task (from live-test, not from this pr)
You can see duplicate fields - I don't know the reason why this is like that originally

![image](https://user-images.githubusercontent.com/17376974/167418773-653565e1-fdbf-4c6e-b827-ec8cd4a346c4.png)

Reply:
One is inside configuration, second one is in the root. This does not look like the same object.

Reply:
So we want it to be inside the configuration and also in the root object? 
Is there any reason?

Reply:
Still not reverted?